.gitmodules

```diff
@@ -275,3 +275,9 @@
 [submodule "libraries/drivers/gpio_expander"]
 	path = libraries/drivers/gpio_expander
 	url = https://github.com/gpongelli/CircuitPython_gpio_expander.git
+[submodule "libraries/helpers/boxplot"]
```

Comment:
It looks like you should remove this first new submodule since it isn't added here.  I'm guessing you renamed it and added it as the submodule below.  I'll take a look at the rest later but wanted to communicate this in the meantime.

Reply:
Yes, having a lot of trouble with this library, so I try to rename it last night.... :( still cannot make it to build, Google and I are out of ideas on how to fix this

```
Error: Error: unexpected status code: 403
{"message":"Resource not accessible by integration","request_id":"XXXXX:XXXXX:XXXXX:XXXXX:XXXXXX,"documentation_url":"https://docs.github.com/rest"}
```

the Xs just in case that number means something. if you know why, please let me know... Thanks :)